setup.cfg

```diff
@@ -13,14 +13,11 @@ parentdir_prefix = pandas-
 
 [flake8]
 ignore =
+    W503,  # line break before binary operator
     E402,  # module level import not at top of file
+    E722,  # do not use bare except
     E731,  # do not assign a lambda expression, use a def
-    W503,  # line break before binary operator
-    C405,  # Unnecessary (list/tuple) literal - rewrite as a set literal.
```

Comment:
are these now checked?

Reply:
Yes, I didn't see any case in the code, so I thought it made sense to start checking them. Any reason I'm missing to keep ignoring them?

Reply:
mhhh, seems like travis and my local flake8 keep checking different things, there wasn't any error in my local call to `lint.sh`, but there are plenty of fails in the travis log. Weird...